### PR TITLE
Split routes and app settings

### PR DESCRIPTION
**Pull request by nicot**
The app configuration (things like port, settings and secrets) should be seperate from routes.

The login route should be at /login.

### CODE REVIEW

File: src/services/db.py [hunk 1]
```diff
@@ -1,16 +0,0 @@
-from remodel.models import Model
```
Comment (antsankov) [inline src/services/db.py:1]: Why is this file being deleted? I thought Sung was working on the DB demo?

File: src/config/config.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+"""
```
Comment (antsankov) [inline src/config/config.py:1]: Good idea including a settings file.

**Comment by antsankov** [comment]
Looks good. Need an explanation for the deleting db.py, but otherwise +1.

File: src/services/db.py (hunk 1 above)
Comment (untra) [inline src/services/db.py:1]: Agreed; this file is currently needed to start the DB. Let @sungbae tackle this.

**Comment by nicot** [comment]
My understanding was that db.py was unused.
All code for the database should be in the models directory. If a service depends on some part of the database, it should import it from models, right?

**Comment by sungbae** [comment]
Yeah I just used db.py as a reference, other than that db.py is pretty much separated inside the models directory as of right now (i.e. section.py, course.py, user.py).

**Pull request merged**